readme.md

```diff
@@ -254,6 +255,19 @@ player.remove(EquipmentSlot, 'rightHand');
 
 ```
 
+#### Cloning components
+
+```js
+const playerA = ecs.createEntity();
+const playerB = ecs.createEntity();
+
+playerA.add(someComponent);
+
+const clonedComponent = player.someComponent.clone();
+
+playerB.attach(clonedComponent); // note the use of `attach` here
```

Comment:
What is the difference here between using attach vs add?

Reply:
The current signature of `add` only allows creating a `new` component, and not adding an existing component instance, and `attach` is the exact opposite. 

I think simplifying it would be a good idea, so `add` would work either way

Reply:
added in #11 `geotic@3.3.0`